### PR TITLE
pkg/loader: Refactor and add tests for translate functions

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

[Related Issue](https://internal.pingcap.net/jira/browse/TOOL-1050)

### What is changed and how it works?

Improve test coverage: 39.1% -> 45.3%

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### CODE REVIEW

File: pkg/loader/translate_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+							},
+						},
+						ChangeRow: &pb.Row{
+							Columns: []*pb.Column{
+								{Int64Value: &oldVal},
+							},
+						},
+					},
+					{
+						Type: pb.MutationType_Insert.Enum(),
```
Comment (csuzhangxc) [inline pkg/loader/translate_test.go]: add case for `DeleteDMLType` and `UnknownDMLType` too?

Comment (suzaku) [reply pkg/loader/translate_test.go]: If we want, we can add those two in tests for `getDMLType`.

**Pull request merged**